week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,10 +10,31 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(listOfBerryColours) {
+  for(let i=0; i < listOfBerryColours.length; i++){
+
+    let berryColour = listOfBerryColours[i];
+    if(berryColour !== "pink") {
+      
+      return "Toxic! Leave bush alone!";}
+    
+}
+  return "Bush is safe to eat from";
 }
 
+console.log(bushChecker(["pink", "pink", "pink", "neon", "pink", "transparent"]))
+console.log(bushChecker(["pink", "pink", "pink", "pink"]))
+// function bushChecker(arr) {
+
+//  if(arr.some(element => element !== "pink")){
```

Comment:
This is really close! Think carefully about what array method we want to use. The goal is to find out if **all** of the items in the array pass the check. (Hint: what's another way of saying "all of the items in the array"?)

Reply:
Another method would be "every".